templates/contactDetails.html

```diff
@@ -42,7 +42,7 @@
 			</div>
 		</header>
 		<section>
-			<propertyGroup ng-repeat="prop in ctrl.contact.props | toArray | orderDetailItems:'$key'" model="ctrl.contact" data="prop" name="prop.$key"></propertyGroup>
+			<propertyGroup ng-repeat="prop in ctrl.contact.props | toArray | orderDetailItems:'$key'" model="ctrl.contact" addressBook="ctrl.addressBook" data="prop" name="prop.$key"></propertyGroup>
```

Comment:
@Henni this doesn't work and fails with a `Error: [$compile:multidir]`
Can you help?

Reply:
Adding the attribute `addressBook` breaks this.
We have a directive which is called `addressBook`, so Angular doesn't know if it should now render the addressBook or the propertyGroup directive.
You have to choose a different attribute name.